src/Views/PartitioningView.vala

```diff
@@ -60,15 +60,16 @@ public class Installer.PartitioningView : AbstractInstallerView  {
 
         load_disks ();
 
-        gparted_button = new Gtk.Button.with_label (_("Modify Partitions"));
+        gparted_button = new Gtk.Button.with_label (_("Modify Partitions…"));
```

Comment:
Opens a new dialog and requires further input to perform the button's action, so we use an ellipsis.

See https://elementary.io/docs/human-interface-guidelines#using-ellipses